src/HotCorners/HotCorner.vala

```diff
@@ -38,25 +38,57 @@ public class Gala.HotCorner : Object {
      * In order to avoid accidental triggers, don't trigger the hot corner until
      * this threshold is reached.
      */
+    private const int TRIGGER_PRESSURE_THRESHOLD_MIN = 0;
     private const int TRIGGER_PRESSURE_THRESHOLD = 50;
 
     /**
      * When the mouse pointer pressures the barrier without activating the hot corner,
      * release it when this threshold is reached.
      */
-    private const int RELEASE_PRESSURE_THRESHOLD = 100;
+    private const int RELEASE_PRESSURE_THRESHOLD_MIN = 30;
+    private const int RELEASE_PRESSURE_THRESHOLD = 70;
 
     /**
      * When the mouse pointer pressures the hot corner after activation, trigger the
      * action again when this threshold is reached.
+     * Only retrigger after a minimum delay (milliseconds) since original trigger.
      */
-    private const int RETRIGGER_PRESSURE_THRESHOLD = 500;
+    private const int RETRIGGER_PRESSURE_THRESHOLD_MIN = 60;
+    private const int RETRIGGER_PRESSURE_THRESHOLD = 440;
+    private const int RETRIGGER_DELAY = 600;
```

Comment:
I wonder if just checking for this delay could be a good solution. Do you think it could work?

Reply:
> I wonder if just checking for this delay could be a good solution. Do you think it could work?

I think so. I don't know why I didn't just try that first. I can test it.